projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.html

```diff
@@ -23,6 +23,9 @@
       <div
         *ngFor="let address of addresses$ | async"
         class="col-md-6 cx-address-card"
+        role="region"
+        tabindex="0"
+        [attr.aria-label]="getRegionLabel(address) | cxTranslate"
```

Comment:
just need to append a number to an additional shipping address label so that it renders "Additional Shipping address 1", "Additional Shipping address 2", etc...

Reply:
Hey @saad-mo,

Yes, I discussed with @bit172 a generic approach to this so that the region and label change is done to the Card component itself and is reusable in any template that uses <cx-card>. This change is being made in PR, #14335.

Will make a few changes to this PR once the other one is merged.